endpoints/put.go

```diff
@@ -87,7 +87,7 @@ func NewPutHandler(backend backends.Backend, maxNumValues int, allowKeys bool) f
 				return
 			}
 
-			logrus.Debugf("Storing value: %s", toCache)
+			//logrus.Debugf("Storing value: %s", toCache)
```

Comment:
Commented out momentarily to not get overwhelmed with log entries

Reply:
I think we should remove this entirely. This would require an insane amount of storage space and I don't see a clear benefit. Reducing this to just IDs should be good enough for those seeking to really trace into exactly what's going on. 

If something is completely broken, we can test from a dev machine with added print line output or with using a debugger.

Reply:
Removed